ClientSide(FrontEnd)/Consumer_Side/src/components/Displayproduct.jsx

```diff
@@ -51,23 +70,23 @@ const Displayproduct = () => {
     <div>
         <div className="color">
         <div className="categories">
-                        <div className="design">
-                            <img src="https://leadwinner.com/shilparamam2/images2/tera/3.jpg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="" />
+                        <div className="design" onClick={() => categoryFunction(116)}>
```

Comment:
every element which can be interacted by the user should have definite html role. you can use semantic html here i.e Button itself